packages/validation/src/__tests__/customSchemas.test.ts

```diff
@@ -0,0 +1,174 @@
+import { expect, it } from "vitest";
+import { Resource } from "../common/types.js";
+import YAML from "yaml";
+
+import "isomorphic-fetch";
+import { createDefaultMonokleValidator } from "../MonokleValidator.js";
+
+it("should work with custom schemas", async () => {
```

Comment:
This PR works for Desktop and Web but it will not make CRDs available for the CLI or GitHub Action.

Not sure where you store the CRDs, but maybe we can move them to the validation configuration? In https://github.com/kubeshop/monokle-core/issues/12 I already briefly mentioned the idea. 

```
settings:
  "custom-schemas":
     "networking.istio.io/v1alpha3-Gateway": "some-http-url-to-schema"
```